tests/test_equal.py

```diff
@@ -0,0 +1,39 @@
+# Licensed under the MIT license.
+
+"""Unit tests for equal"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import tensorflow as tf
+
+from backend_test_base import Tf2OnnxBackendTestBase
+
+
+# pylint: disable=missing-docstring,invalid-name,unused-argument,using-constant-test
+# pylint: disable=abstract-method,arguments-differ
+
+class EqualTests(Tf2OnnxBackendTestBase):
+
+    def test_equal_with_different_parameters(self):
```

Comment:
Put this test in test_backend.py under test_equal.

Reply:
Done :)